### PR TITLE
feat(RHTAPREL-648): update release-service-utils image reference

### PR DESCRIPTION
**Pull request by Troy876**
Release-utils image now refer to redhat-appstudio/release-service-utils

### CODE REVIEW

**Review by johnbieren — COMMENTED** [review]
We need changes to the version strings (the second digit should be bumped) and entries in the README for the changelog per task/pipeline

@Troy876 can you replace all of the fields with `quay.io/redhat-appstudio/release-service-utils:447ea0580a2cdd48b4091e1df86fab5c3f86d01c`?

This is the git sha based tag that we discussed in standup

**Comment by Troy876** [comment]
> We need changes to the version strings (the second digit should be bumped) and entries in the README for the changelog per task/pipeline
> 
> @Troy876 can you replace all of the fields with `quay.io/redhat-appstudio/release-service-utils:447ea0580a2cdd48b4091e1df86fab5c3f86d01c`?
> 
> This is the git sha based tag that we discussed in standup

I've made the appropriate changes, please have a look when you get the chance

File: tasks/collect-data/collect-data.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: collect-data
   labels:
-    app.kubernetes.io/version: "1.0.1"
+    app.kubernetes.io/version: "1.1.1"
```
Comment (johnbieren) [inline tasks/collect-data/collect-data.yaml:7]: This should be 1.1.0

File: tasks/collect-pyxis-params/collect-pyxis-params.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: collect-pyxis-params
   labels:
-    app.kubernetes.io/version: "0.1.1"
+    app.kubernetes.io/version: "0.2.1"
```
Comment (johnbieren) [inline tasks/collect-pyxis-params/collect-pyxis-params.yaml:7]: This should be 0.2.0

File: tasks/publish-pyxis-repository/publish-pyxis-repository.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: publish-pyxis-repository
   labels:
-    app.kubernetes.io/version: "0.0.2"
+    app.kubernetes.io/version: "0.1.2"
```
Comment (johnbieren) [inline tasks/publish-pyxis-repository/publish-pyxis-repository.yaml:7]: This should be 0.1.0

File: tasks/update-infra-deployments/update-infra-deployments.yaml [hunk 4]
```diff
@@ -3,7 +3,7 @@ apiVersion: tekton.dev/v1
 kind: Task
 metadata:
   labels:
-    app.kubernetes.io/version: "0.4.1"
+    app.kubernetes.io/version: "0.5.1"
```
Comment (johnbieren) [inline tasks/update-infra-deployments/update-infra-deployments.yaml:6]: This should be 0.5.0

File: tasks/update-infra-deployments/README.md [hunk 5]
```diff
@@ -18,6 +18,8 @@
 | sharedSecret            | Secret in the namespace which contains private key for the GitHub App                        | true     | infra-deployments-pr-creator                                                                                                                     |
 | githubAppID             | ID of Github app used for updating PR                                                        | true     | 305606                                                                                                                                           |
 | githubAppInstallationID | Installation ID of Github app in the organization                                            | true     | 35269675                                                                                                                                         |
+## Changes since 0.4.1
```
Comment (johnbieren) [inline tasks/update-infra-deployments/README.md:21]: Can you add a new line before this line?

**Review by johnbieren — COMMENTED** [review]
It is comical to me how bad reviewing this would've been if we still had version directories.
Few tiny things to fix, otherwise LGTM

File: tasks/update-infra-deployments/README.md (hunk 5 above)
Comment (Troy876) [reply tasks/update-infra-deployments/README.md:22]: Changes should be visible now

File: tasks/collect-pyxis-params/collect-pyxis-params.yaml [hunk 6]
```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: collect-pyxis-params
   labels:
-    app.kubernetes.io/version: "0.1.1"
+    app.kubernetes.io/version: "0.2."
```
Comment (johnbieren) [inline tasks/collect-pyxis-params/collect-pyxis-params.yaml:7]: forgot the 0

**Review by johnbieren — CHANGES_REQUESTED** [review]

File: tasks/collect-pyxis-params/collect-pyxis-params.yaml (hunk 6 above)
Comment (Troy876) [reply tasks/collect-pyxis-params/collect-pyxis-params.yaml]: Totally intentional
Should be fixed now

**Review by johnbieren — COMMENTED** [review]
LGTM pending testing

**Review by theflockers — DISMISSED** [review]
It feels really nice to have tests for the tasks when we have massive changes like this.

**Comment by Troy876** [comment]
Tested with integration-tests.
All tests pass except rh-push-to-registry-redhat-io, however, current production versions also fails.

File: tasks/push-sbom-to-pyxis/README.md [hunk 7]
```diff
@@ -11,6 +11,9 @@ Tekton task that extracts sboms from pull specs and pushes them to Pyxis.
 | pyxisSecret | The kubernetes secret to use to authenticate to Pyxis. It needs to contain two keys: key and cert | No | - |
 | server | The server type to use. Options are 'production' and 'stage' | Yes | production |
 
+## Chnages since 0.3.1
```
Comment (johnbieren) [inline tasks/push-sbom-to-pyxis/README.md:14]: typo

File: tasks/sign-index-image/README.md [hunk 8]
```diff
@@ -29,5 +29,8 @@ data:
 ## Changes in 1.1.0
 - change the task to use the `internal-request` script
 
+## Changes since 0.2.0
```
Comment (johnbieren) [inline tasks/sign-index-image/README.md:32]: there is something messed up here with the rebase. We shouldn't be adding a changelog entry that isn't the latest. Maybe your PR should now be adding 1.2.0 and an entry for Changes in 1.2.0?

File: tasks/push-sbom-to-pyxis/README.md (hunk 7 above)
Comment (Troy876) [reply tasks/push-sbom-to-pyxis/README.md]: I will never understand how I can continually misspell "changes"
Should be fixed now

**Review by johnbieren — APPROVED** [review]

File: tasks/sign-index-image/README.md [hunk 9]
```diff
@@ -26,6 +26,9 @@ data:
         configMapName: <configmap name>
 ```
 
+## Changes since 1.2.0
```
[the remark this answers is not in the document]
Comment (Troy876) [reply tasks/sign-index-image/README.md]: Fixed
I feel like we should standardise how we record changes, there's like 3 or so different ways to update a README (unless there's a reason for it)

[the remark this answers is not in the document]
Comment (johnbieren) [reply tasks/sign-index-image/README.md]: Agreed, we mentioned this in a standup recently. I'm not sure a ticket was created for it though

**Pull request merged**